src/biogeochem/SatellitePhenologyMod.F90

```diff
@@ -24,7 +24,8 @@ module SatellitePhenologyMod
   private
   !
   ! !PUBLIC MEMBER FUNCTIONS:
-  public :: SatellitePhenology     ! CLMSP Ecosystem dynamics: phenology, vegetation
+  public :: GetSatellitePhenologyInputs ! put the data into the correct format
+  public :: SetSPModeCanopyStructs      ! CLMSP Ecosystem dynamics: phenology, vegetation
```

Comment:
FWIW I find it confusing to refer to CLM Ecosystem Dynamics when this is NOT related to FATES can the comment just be for CLM-SP phenology and vegetation